onnxruntime/core/providers/webgpu/allocator.h

```diff
@@ -25,9 +25,12 @@ class GpuBufferAllocator : public IAllocator {
   virtual void Free(void* p) override;
   void GetStats(AllocatorStats* stats) override;
 
+  Status OnSessionInitializationEnd();
```

Comment:
this function no need to follow the signature of IExecutionProvider::OnSessionInitializationEnd().

for example, it does not need to return a value since the implementation is straightforward enough and cannot fail.

Reply:
Done!